### PR TITLE
Conversation tests

### PR DESCRIPTION
**Pull request by miniwolf**
Sending and receiving message and how they are displayed.
Opening conversations using contact and conversation lists.
Typing text.
Sending using the Enter button and how Shift affects Enter.
etc.

### CODE REVIEW

File: fagiClient/src/com/fagi/controller/conversation/ConversationController.java [hunk 1]
```diff
@@ -9,29 +9,30 @@
...
+import java.util.stream.Collectors;
+
 /**
  * @author miniwolf
  */
 public class ConversationController extends BorderPane {
     @FXML private Label name;
     @FXML private Label date;
-    @FXML private TextArea message;
+    @FXML
```
Comment (Zargess) [inline fagiClient/src/com/fagi/controller/conversation/ConversationController.java]: Here my OCD just screams

File: test/src/com/fagi/guitests/SearchUserTests.java [hunk 2]
```diff
@@ -241,11 +276,11 @@ public void WhenClientShowsSearchResults_ItShouldBeInTheOrderThatTheyWereReceive
         Label label4 = robot.from(contactNodes.get(3)).lookup("#userName").query();
         Label label5 = robot.from(contactNodes.get(4)).lookup("#userName").query();
 
-        Assertions.assertEquals(username1, label1.getText());
-        Assertions.assertEquals(username2, label2.getText());
-        Assertions.assertEquals(username3, label3.getText());
-        Assertions.assertEquals(username4, label4.getText());
-        Assertions.assertEquals(username5, label5.getText());
+        FxAssert.verifyThat(label1, LabeledMatchers.hasText(username1));
```
Comment (Zargess) [inline test/src/com/fagi/guitests/SearchUserTests.java]: Should this be an assertAll?

**Review by Zargess — CHANGES_REQUESTED** [review]
Two minor things, a formating issue that is more my OCD and multiple verifies should be assertAll

File: fagiClient/src/com/fagi/controller/conversation/ConversationController.java (hunk 1 above)
Comment (miniwolf) [reply fagiClient/src/com/fagi/controller/conversation/ConversationController.java]: Fixed https://github.com/miniwolf/Fagi/pull/18/commits/fd80b073521f6d57f05218bd1db2a9f30e943d31

File: test/src/com/fagi/guitests/SearchUserTests.java (hunk 2 above)
Comment (miniwolf) [reply test/src/com/fagi/guitests/SearchUserTests.java]: Fixed https://github.com/miniwolf/Fagi/pull/18/commits/fd80b073521f6d57f05218bd1db2a9f30e943d31

**Review by Zargess — APPROVED** [review]

**Pull request merged**